pkg/kwokctl/runtime/compose/cluster.go

```diff
@@ -504,7 +505,87 @@ func (c *Cluster) Down(ctx context.Context) error {
 	return nil
 }
 
-func (c *Cluster) Start(ctx context.Context, name string) error {
+func (c *Cluster) Start(ctx context.Context) error {
+	conf, err := c.Config(ctx)
+	if err != nil {
+		return err
+	}
+
+	// TODO: nerdctl does not support 'compose start' in v1.1.0 or earlier
+	// Support in https://github.com/containerd/nerdctl/pull/1656 merge into the main branch, but there is no release
+	subcommand := "start"
+	backupFilename := c.GetWorkdirPath("restart.db")
+	if conf.Options.Runtime == consts.RuntimeTypeNerdctl {
+		subcommand = "up"
+	}
+
+	commands, err := c.buildComposeCommands(ctx, subcommand)
+	if err != nil {
+		return err
+	}
+
+	err = exec.Exec(ctx, c.Workdir(), exec.IOStreams{
+		ErrOut: os.Stderr,
+		Out:    os.Stderr,
+	}, commands[0], commands[1:]...)
+	if err != nil {
+		return fmt.Errorf("failed to start cluster: %w", err)
+	}
+
+	if conf.Options.Runtime == consts.RuntimeTypeNerdctl {
+		fi, err := os.Stat(backupFilename)
+		if err != nil && os.IsNotExist(err) {
+			return nil
+		} else if err != nil {
+			return err
+		}
+		if fi.IsDir() {
+			return nil
```

Comment:
This file should never be dir, and I think it should return an error

Reply:
Thanks for tips